Appveyor, MSVC, and QT 5.9.5

Looks like an update to MSVC is creating build errors on MSVC and QT5.9.5
`c:\qt\5.9.5\msvc2015\include\qtcore\qalgorithms.h(847): error C3615: constexpr function 'qCountLeadingZeroBits' `

It appears to only affect the 32bit version of MSVC

This is currently affecting #518 and #522 but in theory will affect all PRs and builds using the 32bit MSVC toolchain.

There are topics about it here:
https://forum.qt.io/topic/93711/problem-with-new-visual-studio-update-2017-version-15-8-0-and-qt-5-9-5/3
http://lists.qt-project.org/pipermail/development/2018-August/033404.html

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -8,8 +8,9 @@ environment:
     PLATFORMTOOLSET: "v140"
   MSBUILD_FLAGS: /verbosity:minimal /maxcpucount
   matrix:
-  - CMAKE_GENERATOR: "Visual Studio 15 2017"
-    QT5: C:\Qt\5.10.1\msvc2015
+# For now, maybe we disable 32 bit build-testing as it's unlikely for our users
+#  - CMAKE_GENERATOR: "Visual Studio 15 2017"
+#    QT5: C:\Qt\5.10.1\msvc2015
   - CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
     QT5: C:\Qt\5.10.1\msvc2017_64
     

